samples/customer-registry-quickstart/README.md

```diff
@@ -0,0 +1,89 @@
+and in particular the [Java section](https://developer.lightbend.com/docs/akka-serverless/java-services/index.html)
+
+
+## Building
+
+To build, at a minimum you need to generate and process sources, particularly when using an IDE.
+To compile your project:
+
+```
+mvn compile
+```
+
+
+## Running Locally
+
+In order to run your application locally, you must run the Akka Serverless proxy. The included `docker-compose` file contains the configuration required to run the proxy for a locally running application.
+It also contains the configuration to start a local Google Pub/Sub emulator that the Akka Serverless proxy will connect to.
+To start the proxy, run the following command from this directory:
+
+
+```
+docker-compose up
+```
+
+
+> On Linux this requires Docker 20.10 or later (https://github.com/moby/moby/pull/40007),
+> or for a `USER_FUNCTION_HOST` environment variable to be set manually.
+
+```
+docker-compose -f docker-compose.yml -f docker-compose.linux.yml up
+```
+
+To start the application locally, the `exec-maven-plugin` is used. Use the following command:
+
+```
+mvn compile exec:java
+```
+
+With both the proxy and your application running, any defined endpoints should be available at `http://localhost:9000`. In addition to the defined gRPC interface, each method has a corresponding HTTP endpoint. Unless configured otherwise (see [Transcoding HTTP](https://developer.lightbend.com/docs/akka-serverless/java/proto.html#_transcoding_http)), this endpoint accepts POST requests at the path `/[package].[entity name]/[method]`. For example, using `curl`:
+
+
+* Create a customer with:
+  ```
+  grpcurl --plaintext -d '{"customer_id": "wip", "email": "wip@example.com", "name": "Very Important", "address": {"street": "Road 1", "city": "The Capital"}}' localhost:9000  customer.api.CustomerService/Create
+  ```
+* Retrieve the customer:
+  ```
+  grpcurl --plaintext -d '{"customer_id": "wip"}' localhost:9000  customer.api.CustomerService/GetCustomer
+  ```
+* Query by name:
+  ```
+  grpcurl --plaintext -d '{"customer_name": "Very Important"}' localhost:9000 customer.view.CustomerByName/GetCustomers
+  ```
+* Change name:
+  ```
+  grpcurl --plaintext -d '{"customer_id": "wip", "new_name": "Most Important"}' localhost:9000 customer.api.CustomerService/ChangeName
+  ```
+* Change address:
+  ```
+  grpcurl --plaintext -d '{"customer_id": "wip", "new_address": {"street": "Street 1", "city": "The City"}}' localhost:9000 customer.api.CustomerService/ChangeAddress
+  ```
+
+## Deploying
+
+To deploy your service, install the `akkasls` CLI as documented in
+[Setting up a local development environment](https://developer.lightbend.com/docs/akka-serverless/getting-started/set-up-development-env.html)
+and configure a Docker Registry to upload your docker image to.
+
+You will need to update the `akkasls.dockerImage` property in the `pom.xml` and refer to
```

Comment:
It looks to me like the property is `dockerImage` not `akkasls.dockerImage`?

Reply:
Yes

(this should probably be updated everywhere)